Core/PhysicsList.cpp

```diff
@@ -23,3 +23,9 @@ PhysicsList::PhysicsList() : G4VModularPhysicsList()
 }
 
 PhysicsList::~PhysicsList() {}
+//Range Cut
+void PhysicsList::SetCuts()
+{
+  //SetVerboseLevel(1);     // seelect verbosity
+  //SetCutValue(1.0 * mm, "e-");  // Production threshold on e- ( must produce with energy to travel 1 mm distance)
```

Comment:
In Geant4 documentation this value is by default set to 1 mm, but I did not find where it is used. What would be the result of making it more or less than 1 mm?

Reply:
The selection of this length(range) trigger the production threshold on the new particle. if the new/secondary particle is able to travel at least the define length, i.e., has sufficient energy to travel that length in chosen material , then only the new particle with produced ( in our case e-)

Reply:
Please refer the two short reports on the energy range cuts:

http://koza.if.uj.edu.pl/petwiki/images/2/24/MC_TimeOptimizationSKS.pdf.    - 12.11.2019
http://koza.if.uj.edu.pl/petwiki/images/0/06/MC_EnergyRangeCuts.pdf           - 16.04.2019

Reply:
Could You also check how the cuts affect the multiplicites of the hits?
i have an impression that it could suppress hits coming from the secondary particles.